www-servers/nginx-unit/nginx-unit-1.10.0.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=(python2_7 python3_{5,6,7})
+
+inherit python-single-r1
+
+MY_P="unit-${PV}"
+DESCRIPTION="A dynamic web and application server"
+HOMEPAGE="https://unit.nginx.org"
+SRC_URI="https://unit.nginx.org/download/${MY_P}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+MY_USE="perl python ruby"
+MY_USE_PHP="php5-6 php7-1 php7-2 php7-3"
+IUSE="${MY_USE} ${MY_USE_PHP}"
+REQUIRED_USE="|| ( ${IUSE} ) python? ( ${PYTHON_REQUIRED_USE} )"
+
+DEPEND="perl? ( dev-lang/perl:= )
+	php5-6? ( dev-lang/php:5.6[embed] )
+	php7-1? ( dev-lang/php:7.1[embed] )
+	php7-2? ( dev-lang/php:7.2[embed] )
+	php7-3? ( dev-lang/php:7.3[embed] )
+	python? ( ${PYTHON_DEPS} )
+	ruby? ( dev-lang/ruby:= )"
+RDEPEND="${DEPEND}"
+S="${WORKDIR}/${MY_P}"
+
+pkg_setup() {
+	use python && python-single-r1_pkg_setup
+}
+
+src_configure() {
+	./configure \
```

Comment:
Why wouldnt `econf` work here? As I see it even gives you some of the defaults youve defined below.

Reply:
NGINX-Unit is a bit unorthodox regarding its build mechanism, and it is necessary to call `configure` multiple times. Once for the core configuration, and once for each of the desired runtime/language modules.